### ISSUE TITLE
Dual, parallel audio playback on system event

### ISSUE THREAD

**Issue by gryzor2** [SkyTubeTeam/SkyTube#1297]
**Describe the bug**
When some Android system event happens, the current playback of a downloaded video starts is restarted. Only one video stream is displayed, but I can hear the audio chanel twice. One is at the current time of the playback at the time of the system event, the other stream is the same video, but started at the start of the video.

**To Reproduce**
1. Download a video
2. Broadcast the video
3. While the video is broadcasting, let a system event (such as "battery is now lower than 25%") occur.
4. Current video keeps playing, but another audio instance of the same video is started in parallel.

Please note that "battery is now lower than 25%" is a real example that triggers the problem, but I'm quite convinced other events can trigger the same misbehavior.

**Expected behavior**
System events should not impact playback this way. No dual broadcasting should ever occur.

**Setup:**
 - This is grapheneOS, stable (build 2024072800 ; Android 14), up to date, running on Pixel 6a
 - Is Your device rooted? Yes
 - Do You use Tor or any VPN service with your device? No.
 - App source  FDroid.
 - App Version 2.989 (2024-07-16 20:22). The bug was already occuring on older versions.
 - App Flavour : I don't know what that means

**Additional context**

**Comment by drogga** [comment]
You are a ver. behind, but I guess given that there are no changed made regarding this, that it will be the same with 2.990.
Anyway, it was fixed before for the streaming, but it appears not for playback of the DLed.

Flavour: Oss or Extra, since you got it from F-Droid > Oss, I recommend switching to the releases from GitHub in future (this won't fix the issue and you will have to do a clean install, an update won't work, because the signatures are different).
